src/test/java/de/hdm_stuttgart/mi/dbad/dbwarp/migration/tablewriter/syntax/column/type/SyntaxColumnTypeMapperTest.java

```diff
@@ -0,0 +1,88 @@
+package de.hdm_stuttgart.mi.dbad.dbwarp.migration.tablewriter.syntax.column.type;
+
+/*-
+ * #%L
+ * DBWarp
+ * %%
+ * Copyright (C) 2024 Kay Knöpfle, Lucca Greschner and contributors
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import de.hdm_stuttgart.mi.dbad.dbwarp.config.Configuration;
+import de.hdm_stuttgart.mi.dbad.dbwarp.model.column.Column;
+import de.hdm_stuttgart.mi.dbad.dbwarp.model.syntax.Syntax;
+import de.hdm_stuttgart.mi.dbad.dbwarp.syntax.loading.SyntaxLoader;
+import java.sql.JDBCType;
+import java.util.Collections;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class SyntaxColumnTypeMapperTest {
+
+  Syntax syntax;
+
+  @BeforeEach
+  void setUp() {
+    Configuration.getInstance().configure(Collections.emptyMap());
+    this.syntax = new SyntaxLoader().loadSyntax("MariaDB");
```

Comment:
Wäre hier eventuell besser, die Syntax zu mocken oder eine spezielle Testsyntax zu laden. Die MariaDB Syntax kann sich potenziell ändern. Außerdem könnten wir mit einer Testsyntax mehr Cases abklappern. Unter Umständen sogar in einem parameterized Test.